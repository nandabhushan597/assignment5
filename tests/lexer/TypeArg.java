x < y
  A<B>
  a<B<C>>
  x <B>
