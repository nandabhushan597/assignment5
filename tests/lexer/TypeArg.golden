Right [IdentifierT "x"
OperatorT LessO
IdentifierT "y"
IdentifierT "A"
SeparatorT LAngleBracketS
IdentifierT "B"
OperatorT GreaterO
IdentifierT "a"
SeparatorT LAngleBracketS
IdentifierT "B"
SeparatorT LAngleBracketS
IdentifierT "C"
OperatorT ShiftRightO
IdentifierT "x"
OperatorT LessO
IdentifierT "B"
OperatorT GreaterO]